server/server.go

```diff
@@ -486,9 +486,6 @@ func (s *Server) ShowProcessList() map[uint64]*util.ProcessInfo {
 	defer s.rwlock.RUnlock()
 	rs := make(map[uint64]*util.ProcessInfo, len(s.clients))
 	for _, client := range s.clients {
-		if atomic.LoadInt32(&client.status) == connStatusWaitShutdown {
-			continue
```

Comment:
Somebody once complained that he use `kill tidb connection XXX`, but could still see the session in `show processlist`.
Here is a fix for him.

Reply:
I think it should stay in `processlist` because it is really running and not fully closed. Maybe we can show a `closing` state for hime.